src/MembershipsSDKAdapter.js

```diff
@@ -39,8 +56,8 @@ const MAX_MEMBERSHIPS = 1000;
  * @returns {Array} List of active users in a meeting
  */
 function getMembers(members) {
-  return members ? Object.values(members)
-    .filter((member) => member.isUser)
+  return members ? sortMembers(Object.values(members)
```

Comment:
I would suggest this change, to make the data flow a little easier to follow:
```js
members = Object.values(members || {});
members = members.filter((member) => member.isUser);
members = sortMeetingMembers(members);
return members.map(...);
```

this way, the processing steps on the members are listed in chronological order.